src/Core/OrganizationFeatures/AuthorizationHandlers/GroupOperations.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bit.Core.OrganizationFeatures.AuthorizationHandlers;
+
+public class GroupOperationRequirement : OperationAuthorizationRequirement { }
+
+public static class GroupOperations
+{
+    // Operations on the Group object itself
+    public static readonly GroupOperationRequirement Create = new() { Name = "Create" };
+    public static readonly GroupOperationRequirement Read = new() { Name = "Read" };
+    public static readonly GroupOperationRequirement Update = new() { Name = "Update" };
+    public static readonly GroupOperationRequirement Delete = new() { Name = "Delete" };
+
+    // Operations on Group-User associations 
+    public static readonly GroupOperationRequirement AddUser = new() { Name = "AddUser" };
+    public static readonly GroupOperationRequirement DeleteUser = new() { Name = "DeleteUser" };
```

Comment:
Personally I'd say these should fall under their own "Operations" class. Would we ever want there to be `AddGroup` and `DeleteGroup` on `UserOperations` if no, and those would be the same permission I'd rather these be `GroupUser.Delete` and `GroupUser.Create`.

Reply:
Yes, I think these are operations on the `GroupUser` rather than the `Group`. We have a `GroupUser` object that we can pass as a resource instead, and those checks can be in their own handler.

Reply:
`GroupUser` has now been implemented as a separate resource.